.travis.yml

```diff
@@ -4,6 +4,7 @@ go_import_path: github.com/colinmarc/hdfs
 go: 1.x
 env:
 - PLATFORM=cdh5
+- PLATFORM=cdh5 KERBEROS=true
 - PLATFORM=hdp2
```

Comment:
Why are we not testing `KERBEROS=true` for `hdp2` ?

Reply:
Because it's redundant, and because I didn't factor out the krb setup stuff (although I could probably). I don't actually think it's really necessary to have `hdp2` in there right now, but I do want to have `hdp3` shortly (I think I need xenial travis support for that?) and `cdh6` eventually.